common/src/main/java/gov/cms/ab2d/common/repository/CoverageRepository.java

```diff
@@ -22,4 +23,28 @@
 
     Optional<Coverage> findByContractAndBeneficiaryAndPartDMonth(Contract contract, Beneficiary bene, int month);
 
+
+
+    @Modifying
+    @Query(" DELETE FROM Coverage c  ")
+    int deleteAllInBulk();
```

Comment:
`deleteAll`?

Reply:
This is the signature for delete all 
```
   void deleteAll();
```

It all does not return the number of rows deleted. I was logging the number of rows deleted for the other calls, so opted to be consistent with that.  But I could change it to the deleteAll() method for this case.

Reply:
done - changed to use `deleteAll()`